packages/frontend/apps/electron/src/main/ui/handlers.ts

```diff
@@ -220,4 +221,14 @@ export const uiHandlers = {
     app.relaunch();
     app.quit();
   },
+  onLanguageChange: async (e, language: string) => {
+    // only works for win/linux
```

Comment:
any reason or read more link?

Reply:
I find it it electron docs, better quote it https://www.electronjs.org/docs/latest/tutorial/spellchecker#how-to-set-the-languages-the-spellchecker-uses